### ISSUE TITLE
run get-reads-alt-unmap.sh in container

### ISSUE THREAD

**Issue by tmhmxg59** [humanlongevity/HLA#35]
I'm struggling to get typing information from hg19 aligned bams.  I think I'd like to run get-reads-alt-unmap.sh in the docker container - I don't have all the dependencies on my machine.

I think I need to change the entrypoint, but I'm not sure the syntax.  The coordinates in the .sh script appear to cover all genes in both references.

I tried several variations of:
$docker run -itv `pwd`:`pwd` `pwd` -w humanlongevity/hla --entrypoint /bin/bash ls
docker: invalid reference format: repository name must be lowercase.
See 'docker run --help'.

But doesn't seem like I've successfully changed the entrypoint.

**Comment by adeschen** [comment]
Hi, 

xHLA only works with hg38 reference. 

The get-reads-alt-unmap.sh script needs to be used when you have a BAM aligned on hg38 reference with alt contigs.

Best,
Astrid

**Comment by tmhmxg59** [comment]
Conversion from 37 to 38 worked for me for one bam.  I used:
docker run -v ~/bams:/opt/bin/myData -it --entrypoint=/bin/bash humanlongevity/hla
to open a container with my local directory /bams bound to /opt/bin/myData.

I did have to remove the memory detection part of the script as it gave an error at line 17.
And add the fna as @cjieming noted.

On a second data set, though, I'm loosing about half of the reads somewhere in the script.  My first go I added a few reads as expected.  The only thing I can think of is: when I ran the first time I was using a 37 bam PrintRead to chr6.  The first step in the script is to trim, though, so I'm not sure what the issue is.
